src/adapters/manta.ts

```diff
@@ -86,6 +94,7 @@ export const calamariTokensConfig: Record<string, BasicToken> = {
   LKSM: { name: "LKSM", symbol: "LKSM", decimals: 12, ed: "500000000" },
   KSM: { name: "KSM", symbol: "KSM", decimals: 12, ed: "100000000" },
   MOVR: { name: "MOVR", symbol: "MOVR", decimals: 18, ed: "100000000000000000" },
+  USDT: { name: "USDT", symbol: "USDT", decimals: 6, ed: "10000" }
```

Comment:
I think decimals of USDT on Statemine is 3, not 6.
<img width="629" alt="image" src="https://user-images.githubusercontent.com/1088525/228837283-7b861683-b739-44fc-9f33-67c756d6caef.png">